.travis.yml

```diff
@@ -4,7 +4,8 @@ services:
   - postgresql
 
 before_script:
-  - psql -U postgres -c 'CREATE DATABASE chitter_test;'
+  - psql -U postgres -c 'CREATE DATABASE chitterboard_test;'
+  - psql --dbname=chitterboard_test -U postgres -c 'CREATE TABLE cheets(id SERIAL PRIMARY KEY, message VARCHAR(140), display_name VARCHAR(60), time_stamp timestamp default now());'
```

Comment:
I missed adding the line 8 table creation to this file. Well done for adding it!